tg_bot/modules/log_channel.py

```diff
@@ -44,29 +45,55 @@ def log_action(bot: Bot, update: Update, *args, **kwargs):
     def log_resource(bot: Bot, update: Update):
         entities = update.effective_message.parse_entities()
         caption_entities = update.effective_message.parse_caption_entities()
+        LOGGER.info(entities)
         chat = update.effective_chat  # type: Optional[Chat]
         log_chat = sql.get_chat_log_channel(chat.id)
+
+        url = None
+        tags = []
+
         if log_chat:
-            result = f'<b>Risorsa inviata da @{update.effective_user.username}:</b>\n'
+            for descriptor, entity in entities.items():
+                if descriptor['type'] == MessageEntity.HASHTAG:
```

Comment:
Questo controllo è possibile farlo nel for loop della riga 62 ? visto che è lo stesso identico ciclo, almeno risparmiamo un'iterazione